Rubberduck.Core/AddRemoveReferences/ReferenceModel.cs

```diff
@@ -83,10 +99,21 @@ public ReferenceModel(ITypeLib reference)
             }
         }
 
-        public ReferenceModel(string path)
+        public ReferenceModel(string path, bool broken = false) : this()
         {
             FullPath = path;
-            Name = Path.GetFileName(path);
+            try
+            {
+                Name = Path.GetFileName(path);
+                Description = Name;
+            }
+            catch
+            {
+                // Yeah, that's probably busted.
+                IsBroken = true;
+            }
+            
+            IsBroken = broken;
```

Comment:
This looks funny. In the `catch` we set it to `True` - wouldn't that then get overwritten here?

Reply:
Oops. That should have a return.

Reply:
Is this outdated/resolved?

Reply:
@retailcoder  Yep.